build/documentation/copy-package-readme.js

```diff
@@ -8,15 +8,19 @@
 const path = require("path");
 const fs = require("fs");
 const glob = require("glob");
+const chalk = require('chalk');
 
 const rootDir = path.resolve(process.cwd());
-const srcReadmePaths = "packages/*/README.md";
+const srcReadmePaths = "packages/*/?(readme.md|README.md)";
 const destDir = path.join("docs", "en", "packages");
 const srcSidebar = path.join("website", "sidebars.json");
 
 var dryRun = false;
 var sidebarEntries = [];
 
+var totalDocs;
+var updatedDocs = 0;
```

Comment:
Maybe name this `updatedDocsCount` so that this variable reads more like a counter for how many documents have been updated and not a container for the updated documents themselves.